src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.AnalyzerExecutionContext.cs

```diff
@@ -186,7 +188,7 @@ void processMembers(IEnumerable<ISymbol> members)
                         {
                             if (!member.IsImplicitlyDeclared && member.IsInSource())
                             {
-                                memberSet = memberSet ?? new HashSet<ISymbol>();
+                                memberSet ??= new HashSet<ISymbol>();
```

Comment:
📝 This could have been a separate change

Reply:
Yes, but these are very minor code cleanup fixes. I don't see a value in moving these out unless they are a review hindrance.

Reply:
I would agree with Manish. My main concern with this PR is the number of files, not suffix changes/clarity refactorings that (at least for me) aren't a hinderance.

---
In reply to: [467129196](https://github.com/dotnet/roslyn/pull/46409#discussion_r467129196) [](ancestors = 467129196)